apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -52,32 +52,81 @@ When considering compute upgrades, assess whether your bottlenecks are hardware-
+| 2XL         | 4,750 Mbps          | 2,375 Mbps               | 20,000 IOPS | 12,000 IOPS   |
+| 4XL         | 4,750 Mbps          | 4,750 Mbps               | 20,000 IOPS | 20,000 IOPS   |
+| 8XL         | 9,500 Mbps          | 9,500 Mbps               | 40,000 IOPS | 40,000 IOPS   |
+| 12XL        | 14,250 Mbps         | 14,250 Mbps              | 50,000 IOPS | 50,000 IOPS   |
+| 16XL        | 19,000 Mbps         | 19,000 Mbps              | 80,000 IOPS | 80,000 IOPS   |
+
+### Self-Serve Configurability
+
+You can now adjust storage type, IOPS and disk size parameters yourself through the Dashboard [Database Settings page](https://supabase.com/dashboard/project/_/settings/database).
+
+![Disk management configuration](/docs/img/guides/platform/disk-mgmt-io.png)
+
+<Admonition type="caution">
+
+Increasing IOPS or throughput incurs additional charges.
+
+</Admonition>
+
+### Understanding I/O Balance and Bursting (Smaller Instances)
+
+For smaller instances like Nano, Micro, and Small, you get I/O balance and burst capacity. These instances have limited base performance for IOPS and throughput. However, they come with burst credits that let you handle temporary spikes for 30 minutes in a day on demand by briefly exceeding their baseline performance. This means you can achieve higher performance for short bursts before reverting to your baseline limits.
+
+For example, the micro plan can burst up to 2,085 Mbps for 30 minutes a day and reverts to the baseline performance of 87 Mbps. Your disk IO budget gets replenished throughout the day.
+
+### Dedicated I/O Resources (Larger Instances)
+
+Larger instances, such as 4XL and above, come with dedicated I/O resources and don't rely on I/O balance or burst capacity. These instances are designed for sustained, high performance with specific IOPS and throughput limits. They provide consistent performance without needing to burst. If you hit your IOPS or throughput limit, throttling will occur, but you won't have burst credits to exceed these limits.
+
+### Storage Volume Types: gp3 vs. io2
+
+Here’s how **gp3** and **io2** storage volumes differ:
+
+**gp3 (General Purpose SSD) (default)** This is a versatile and cost-effective option, perfect for a range of everyday workloads that need moderate IOPS and throughput. It supports volume sizes from 1GB to 16TB. You get IOPS ranging from 3,000 (default) up to 16,000 (maximum), and throughput from 125 MB/s (default) to 1,000 MB/s (maximum). It’s well-suited for general tasks like boot volumes and small to medium-sized databases.
+
+**io2 (Provisioned IOPS SSD)** Designed for high-demand scenarios where very high IOPS and low latency are crucial. It supports volumes ranging from 4GB to 16TB. With IOPS reaching up to 256,000 (depending on your instance type) and throughput scaling with IOPS, io2 is designed for large-scale databases and mission-critical applications. While it offers top-tier performance, it does come with a higher price tag.
```

Comment:
@delgado3d what did we decide was the limit, 60TB for now? Or something lower like 30TB and we can increase as needed?

Reply:
yeah we decided 60 TB, which should still give us a bit of a buffer